sparkmagic/sparkmagic/magics/sparkmagicsbase.py

```diff
@@ -37,6 +39,29 @@ def __init__(self, shell, data=None, spark_events=None):
             spark_events = SparkEvents()
         spark_events.emit_library_loaded_event()
 
+    def execute_local(self, cell, input_variable_name, type, output_variable_name, session_name):
+        input_variable_value = self.shell.user_ns[input_variable_name]
+
+        if not input_variable_value:
+            raise BadUserDataException("Value of specified -o variable is None!")
+
+        if not output_variable_name:
+            output_variable_name = input_variable_name
+
+        if type == 'str':
+            command = SendStringToSparkCommand(input_variable_name, input_variable_value, output_variable_name)
+        elif type == 'df':
+            command = SendPandasDfToSparkCommand(input_variable_name, input_variable_value, output_variable_name)
+        else:
+            raise BadUserDataException("Invalid -t type. Available are: `str` or `df`.")
```

Comment:
Let's construct the available types by joining an array from constants

Reply:
will do